hack/generate-latest-csv.sh

```diff
@@ -49,4 +49,5 @@ $CSV_CHECKSUM \
 	--noobaa-core-image="$NOOBAA_CORE_IMAGE" \
 	--noobaa-db-image="$NOOBAA_DB_IMAGE" \
 	--ocs-image="$OCS_IMAGE" \
+	--ocs-must-gather-image="$MUST_GATHER_FULL_IMAGE_NAME" \
```

Comment:
I would follow the pattern of adding a variable `MUST_GATHER_IMAGE` to this script and set it by default to "$MUST_GATHER_FULL_IMAGE_NAME" (in the init section above). This would allow for overriding. Not strictly needed, but nice and consistent.